frame/support/src/instances.rs

```diff
@@ -0,0 +1,90 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Some instance placeholder to be used in `pallet` attribute macro.
+//!
+//! `pallet` attribute macro do not make use of `I` instance generic for anything, thus the
```

Comment:
I don't understand what you mean by this.

Reply:
I wanted to emphasize that having instance defined here is possible because pallet doesn't implement information on them.
Old pallet still define their own instance of `Instance1`..`InstanceN` because they need to implement the trait `support::traits::Instance` on them to give the full module prefix to use.

But I'll try to come up with better sentence here

Reply:
I rephrased it, should be better now